test/performance/Benchmarks.md

```diff
@@ -0,0 +1,62 @@
+# Benchmarks
+
+Knative performance benchmarks are tests geared towards producing useful
+performance metrics of the knative system. All the raw metrics are stored in
+[mako](https://github.com/google/mako)
+
+## Existing scripts
+
+There are two existing scripts that help in recycling the benchmark clusters
+and have latest knative/serving updates:
+
+1. [update.sh](https://github.com/knative/serving/blob/master/test/performance/tools/update-serving): Update all the benchmark clusters every hour
+2. [recreate.sh](https://github.com/knative/serving/blob/master/test/performance/tools/recreate-serving): Re-create the benchmark cluster once a day
+
+## Writing new benchmarks
+
+For creating new benchmarks, follow the steps:
+
+1. Create a new directory under `./test/performance/`. The benchmark clusters 
+2. Create a new [mako config](https://github.com/google/mako/blob/github-push-test-1/docs/GUIDE.md#preparing-your-benchmark).
+   Create a new benchmark using that config.
+3. Create a new knative service(if any) that the benchmark will use to test the knative components.
+4. Create other k8s objects(if any) needed for the benchmarks
+5. Write a go file that will run the benchmarks and store results in [mako](##Writing-to-mako)
+6. Create a cronjob that will run the benchmark at some frequency. Add the mako microservice as a sidecar and add the robot account
+7. Create a symlink to HEAD `ln -s -r .git/HEAD ./test/performance/cmd/<dir>/kodata/`
+8. Run the [create_cluster_benchmark.sh](https://github.com/knative/serving/blob/master/test/performance/tools/create_cluster_benchmark.sh)
+script as
+
+```bash
+./create_cluster_benchmark.sh --name=<dir_name> --zone=<zone> --num_nodes=<node-count>
```

Comment:
Is there a way to run this with your own project?

Reply:
Currently, no. But, we can add that.

Reply:
Is it `--zone`?  I thought we used regional clusters?

Reply:
I feel like the title and content of this page disagree on their audience.

This script can be run by a handful of people, but the audience of "how to write a benchmark" should be effectively the entire community.  Can we approach this doc from the perspective of a community member who doesn't have access to knative-performance and how they'd go about authoring a new benchmark and then getting it into our automation?